config/boards/yy3568.csc

```diff
@@ -0,0 +1,12 @@
+# Rockchip RK3568 quad core 1-8GB SoC 2x1GBe eMMC USB3
+BOARD_NAME="yy3568"
```

Comment:
Perhaps here you could use the full board name `YouYeeToo YY3568", as it is supposed to contain the full manufacturer +  friendly name of the board:

```
paolo@armbian-build:~/armbian-build/config/boards$ rgrep BOARD_NAME
nanopi-r5s.csc:BOARD_NAME="NanoPi R5S"
orangepi.eos:BOARD_NAME="Orange Pi"
hinlink-h66k.csc:BOARD_NAME="Hinlink H66K"
khadas-vim3l.conf:BOARD_NAME="Khadas VIM3L"
rockpi-e.conf:BOARD_NAME="Rockpi E"
...
```